pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
@bparees now that it's morning here and I can function normally: the reason the pastebin I posted seems like working is because when running with `--from-build` specified, it's essentially re-running a previous build so `buildName` is the name of the build in that particular case.